elide-core/src/main/java/com/yahoo/elide/core/exceptions/IllegalPermissionExpressionException.java

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright 2016, Yahoo Inc.
+ * Licensed under the Apache License, Version 2.0
+ * See LICENSE file in project root for terms.
+ */
+package com.yahoo.elide.core.exceptions;
+
+import com.yahoo.elide.core.HttpStatus;
+
+/**
+ *
+ * Exception showing invalid combination of FilterExpressionCheck and other checks.
+ *
+ */
+public class IllegalPermissionExpressionException extends HttpStatusException {
+
+    public IllegalPermissionExpressionException(String left, String right) {
+        super(HttpStatus.SC_BAD_REQUEST, "Invalid Permission Expression: '" + left + "' with " + "'" + right + "'");
```

Comment:
An invalid permission expression doesn't have anything to do with a bad client request, but rather bad server config.  We should probably just remove this exception and throw an IllegalStateException instead.